mdl/src/main/cft/mdlElasticsearch.yml

```diff
@@ -334,6 +334,86 @@ Resources:
     Properties:
       Handle: !Ref EsWaitHandle
       Timeout: '600'
+  LambdaExecutionRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Path: /
+      Policies:
+        - PolicyName: root
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                  - logs:CreateLogGroup
+                  - logs:CreateLogStream
+                  - logs:PutLogEvents
+                Resource: arn:aws:logs:*:*:*
+              - Effect: Allow
+                Action:
+                  - acm:DeleteCertificate
+                  - ssm:GetParameter
+                Resource: '*'
+  CleanupAcmLambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Handler: 'index.handler'
+      Code:
+        ZipFile: |
+          import cfnresponse
+          import boto3
+
+          def handler(event, context):
+              instance_name = event['ResourceProperties']['MDLInstanceName']
+              environment = event['ResourceProperties']['Environment']
+              auth_enabled = event['ResourceProperties']['EnableSSLAndAuth']
+              physical_id = "CleanupAcmCertLambdaPhysicalID"
+              response_data = {}
+              if event['RequestType'] == 'Delete' and auth_enabled == 'true':
+                  try:
+                      client = boto3.client('acm')
+                      print('Deleting acm certificate')
+                      cert_ssm_key = construct_acm_ssm_key(instance_name, environment)
+                      certificate_arn = get_ssm_string(cert_ssm_key)
+                      response = client.delete_certificate(
+                          CertificateArn=certificate_arn
+                      )
+                      response_data['Message'] = 'Successfully deleted ACM cert with ARN'.format(certificate_arn)
```

Comment:
This needs a`{}` in the string to perform the positional replacement.

Reply:
this explains why this print out is not displayed even though the certs got deleted successfully.